api/src/main/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimController.java

```diff
@@ -73,7 +76,11 @@ private VoucherEntitlementDTO getEntitlement(ClaimResult result) {
         return result.getVoucherEntitlement().map(voucherConverter::convert).orElse(null);
     }
 
-    private HttpStatus getHttpStatus(ClaimStatus claimStatus) {
+    private HttpStatus getHttpStatus(ClaimResult claimResult) {
+        if (TRUE.equals(claimResult.getClaimUpdated())) {
```

Comment:
I know this is correct, but it looks really ugly, can we have an `isClaimUpdated()` method on `ClaimResult` so this can just be `if(claimResult.isUpdated())`, which would look and read much better.

Reply:
As a compromise, I've updated to use isTrue from apache commons.